public/js/tracking_protection.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-env browser */
+"use strict";
+
+/* 
+	Respect user privacy settings and honor user privacy choices.
+	From Schalk Neethling at https://github.com/schalkneethling/dnt-helper */
```

Comment:
Random thought, please feel free to hurl profanity at me in any channel you want, but...
Would it make more sense to include this dnt-helper 3rd party code via npm instead of copy/pasta?

Well, technically Schalk Neethling hasn't published to npm, but did generously put a package.json file in the repo, so we should be able to include this via GitHub using:

```sh
$ npm i schalkneethling/dnt-helper -S
```

Then we'd just have to copy the file from ./node_modules/ using something adjacent to this:

```js
$ cp ./node_modules/dnt-helper/js/dnt-helper.js public/js/vendor
```
We could even wrap that in some "postinstall" npm script, which copies that one vendor file into the correct location. Or, maybe this is well overkill and we revisit this after Shield launch.

Reply:
No I like this. It feels bonafide.

Reply:
Going to merge as is for now and add this as an issue.